## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 433936`
Commit: `b4395380aa5f7f2afc03efa2fd04610f10317e76`

---
### `x86_64-linux`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>python312Packages.brax</li>
    <li>python312Packages.brax.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 52 packages built:</summary>
  <ul>
    <li>python312Packages.ale-py</li>
    <li>python312Packages.ale-py.dist</li>
    <li>python312Packages.arviz</li>
    <li>python312Packages.arviz.dist</li>
    <li>python312Packages.bambi</li>
    <li>python312Packages.bambi.dist</li>
    <li>python312Packages.corner</li>
    <li>python312Packages.corner.dist</li>
    <li>python312Packages.dalle-mini</li>
    <li>python312Packages.dalle-mini.dist</li>
    <li>python312Packages.evosax</li>
    <li>python312Packages.evosax.dist</li>
    <li>python312Packages.flax</li>
    <li>python312Packages.flax.dist</li>
    <li>python312Packages.flowjax</li>
    <li>python312Packages.flowjax.dist</li>
    <li>python312Packages.gymnasium</li>
    <li>python312Packages.gymnasium.dist</li>
    <li>python312Packages.minari</li>
    <li>python312Packages.minari.dist</li>
    <li>python312Packages.numpyro</li>
    <li>python312Packages.numpyro.dist</li>
    <li>python312Packages.nutpie</li>
    <li>python312Packages.nutpie.dist</li>
    <li>python312Packages.orbax-checkpoint</li>
    <li>python312Packages.orbax-checkpoint.dist</li>
    <li>python312Packages.pettingzoo</li>
    <li>python312Packages.pettingzoo.dist</li>
    <li>python312Packages.pymc</li>
    <li>python312Packages.pymc.dist</li>
    <li>python312Packages.shimmy</li>
    <li>python312Packages.shimmy.dist</li>
    <li>python312Packages.skrl</li>
    <li>python312Packages.skrl.dist</li>
    <li>python312Packages.stable-baselines3</li>
    <li>python312Packages.stable-baselines3.dist</li>
    <li>python312Packages.tianshou</li>
    <li>python312Packages.tianshou.dist</li>
    <li>python312Packages.tyro</li>
    <li>python312Packages.tyro.dist</li>
    <li>python312Packages.unsloth</li>
    <li>python312Packages.unsloth-zoo</li>
    <li>python312Packages.unsloth-zoo.dist</li>
    <li>python312Packages.unsloth.dist</li>
    <li>python312Packages.viser</li>
    <li>python312Packages.viser.dist</li>
    <li>python312Packages.vqgan-jax</li>
    <li>python312Packages.vqgan-jax.dist</li>
    <li>python312Packages.waymax</li>
    <li>python312Packages.waymax.dist</li>
    <li>python313Packages.orbax-checkpoint</li>
    <li>python313Packages.orbax-checkpoint.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python312Packages.brax</summary>
<pre>brax/training/agents/ars/train_test.py::ARSTest::testModelEncoding0
  /nix/store/9fz60xkkkqny8iwwy2mmr1wra2k1a7zd-python3.12-jax-0.6.2/lib/python3.12/site-packages/jax/_src/numpy/reductions.py:1250: DeprecationWarning: jnp.std: where must be None or a boolean array; got dtype=int32.
    where = check_where("std", where)

brax/mjx/perf_test.py::PerfTest::test_pipeline_ant
brax/mjx/pipeline_test.py::PipelineTest::test_pendulum
brax/mjx/pipeline_test.py::PipelineTest::test_pipeline_init_with_ctrl
  /build/source/brax/mjx/pipeline.py:77: DeprecationWarning: Accessing `contact` directly from `Data` is deprecated. Access it via `data._impl.contact` instead.
    brax_contact = _reformat_contact(sys, data.contact)

brax/mjx/perf_test.py::PerfTest::test_pipeline_ant
brax/mjx/pipeline_test.py::PipelineTest::test_pendulum
  /build/source/brax/mjx/pipeline.py:112: DeprecationWarning: Accessing `ncon` directly from `Data` is deprecated. Access it via `data._impl.ncon` instead.
    if data.ncon > 0:

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED brax/training/agents/ppo/checkpoint_test.py::CheckpointTest::test_save_and_load_checkpoint - ValueError: Expected list, got RestoreArgs(restore_type=<class 'numpy.ndarr...
FAILED brax/training/agents/sac/checkpoint_test.py::CheckpointTest::test_save_and_load_checkpoint - ValueError: Expected list, got RestoreArgs(restore_type=<class 'numpy.ndarr...
=========== 2 failed, 328 passed, 356 warnings in 536.58s (0:08:56) ============</pre>
</details>
</details>

--- a/pkgs/development/python-modules/brax/default.nix
+++ b/pkgs/development/python-modules/brax/default.nix
@@ -38,14 +38,14 @@
 
 buildPythonPackage rec {
   pname = "brax";
-  version = "0.12.4";
+  version = "0.13.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "google";
     repo = "brax";
     tag = "v${version}";
-    hash = "sha256-/eb0WjMzHwD1tjTyZ2fb2dzvGrWnyOLcVLOx4BeKvqk=";
+    hash = "sha256-mSFbFzSrfAvAE6y7atUeucUkpp/20KP70j5xPm/xvB0=";
   };
 
   build-system = [

